src/System.Management.Automation/engine/CmdletInfo.cs

```diff
@@ -68,6 +68,9 @@ public class CmdletInfo : CommandInfo
             _helpFilePath = helpFile;
             _PSSnapin = PSSnapin;
             _options = ScopedItemOptions.ReadOnly;
+
+            // CmdletInfo represents cmdlets exposed from assemblies, and on a DeviceGuard machine
```

Comment:
The comment should probably say "on a system locked down machine" because there are other ways to lock down a machine including AppLocker or third part application white listing solutions.